apps/pie-docs/src/patterns/haptic-feedback/guidance/guidance.md

```diff
@@ -0,0 +1,139 @@
+        "**Gesture recognition:** Use medium haptic feedback to confirm the recognition of gestures, such as pinch-to-zoom or swipe gestures."
+    ]
+} %}
+
+---
+
+## Success haptic feedback
+
+A medium-intensity haptic feedback used to highlight a confirmation of an action. You can use it for the successful submission of a form or completion of a task, providing users with a reassuring response.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/success-haptic-feedback.svg",
+  alt: "Two vertical orange bars, one short the next long, placed side by side and intersected by a horizontal line, representing success haptic feedback signals.",
+  width: "344px"
+} %}
+
+#### Where can I use success haptic feedback?
+
+{% list {
+    type: listTypes.icon,
+    iconName: "check-circle",
+    items: [
+        "**Task completion:** Use success haptic feedback to celebrate the successful completion of a task or operation, providing positive reinforcement.",
+        "**Confirmation of positive events:** Accompany positive events, such as successful transactions or the completion of a level in a game, with success haptic feedback.",
+        "**Goal achievement:** Celebrate users reaching specific milestones or goals within an app or gamified experience with success haptic feedback."
+    ]
+} %}
+
+---
+
+## Warning haptic feedback
+
+They can be strategically used in various scenarios where it's essential to alert users to critical information or potential issues that demand their attention before they actually happen.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/warning-haptic-feedback.svg",
+  alt: "Two vertical orange bars, one longer, the next short, placed side-by-side and intersected by a horizontal line, representing warning haptic feedback signals.",
+  width: "344px"
+} %}
+
+#### Where can I use warning haptic feedback?
+
+{% list {
+    type: listTypes.icon,
+    iconName: "check-circle",
+    items: [
+        "**Alerts and warnings:** Use warning haptic feedback to highlight critical information or alerts that require users' attention but are not urgent.",
+        "**Potential issues:** Signal potential issues or warnings that users need to be aware of, such as incomplete form entries or system warnings.",
+        "**Near limits:** Provide warning haptic feedback when users are approaching or exceeding certain limits, such as character count or storage capacity."
+    ]
+} %}
+
+---
+
+## Error haptic feedback
+
+A strong, noticeable vibration. You can use them when an error occurs or an invalid action is attempted, signalling to the user that corrective action is needed.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/error-haptic-feedback.svg",
+  alt: "Four vertical orange bars, the third one longer than the others, placed side by side and intersected by a horizontal line, symbolizing error haptic feedback signals.",
```

Comment:
Actually I'm starting to wonder if it's best to just describe the haptic feedback pattern, i.e., four vibrations with the third being stronger, and the fourth longer (is this right?), rather than describing the image itself.

Reply:
@xander-marjoram mb will be better to get descriptions from official resources https://developer.apple.com/design/human-interface-guidelines/playing-haptics?

- Soft notification haptics indicates a collision between soft or flexible UI objects.
- Medium notification haptics indicates a collision between medium-sized or medium-weight UI objects.
- Success notification haptics indicates that a task or action has completed.
- Warning notification haptics indicates a task or action has produced a warning of some kind.
- Error notification haptics indicates that an error has occurred.

Reply:
@maledr5 what do you think about these alt texts? Can I leave them as it is here? or will be better to change smth?

> mb will be better to get descriptions from official resource https://developer.apple.com/design/human-interface-guidelines/playing-haptics#Notification?
> 
> * Soft notification haptics indicates a collision between soft or flexible UI objects.
> * Medium notification haptics indicates a collision between medium-sized or medium-weight UI objects.
> * Success notification haptics indicates that a task or action has completed.
> * Warning notification haptics indicates a task or action has produced a warning of some kind.
> * Error notification haptics indicates that an error has occurred.

Reply:
Hey, @xander-marjoram @leksaBoiko what do you think if we leave the text as images descriptors for now and bring the topic to the a11y guild, maybe Geri and Maryia have some knowledge of what is a good practice. I don't want to block this story so I am happy to open that discussion in the a11y guild and make changes after if any comes up.